Makefile

```diff
@@ -63,7 +69,14 @@ test-all: babelify
 		--bail \
 		--reporter spec
 
+commit-dist:
+	make clean;
+	make;
```

Comment:
>Yes, because `make clean` removes all compiled files into `dist/` folder. Compile and update source files is a recurrent task so I've created this `commit-dist` rule.

So the `make;` command is essentially an alias for `standalone`? Is this only by consequence of it occurring first in the Makefile (showing my own ignorance of Make here). Would it be better to call `make standalone;` explicitly?